CHANGES.rst

```diff
@@ -187,6 +193,10 @@ astropy.samp
 astropy.stats
 ^^^^^^^^^^^^^
 
+- The ``BoxLeastSquares`` and ``LombScargle`` classes have been moved to the
```

Comment:
add an entry about the move under `astropy.timeseries`, too.

Reply:
Done

Reply:
In the interest of saving CI time, I'll add it in https://github.com/astropy/astropy/pull/8599 if that's ok?